DigitalLearningSolutions.Web/Views/TrackingSystem/CourseSetup/AddNewCentreCourse/_SelectCourseFilterableSelectListItem.cshtml

```diff
@@ -1,9 +1,12 @@
 ﻿@using DigitalLearningSolutions.Web.ViewModels.TrackingSystem.CourseSetup.AddNewCentreCourse
 @model FilterableApplicationSelectListItemViewModel
 
-<option value="@Model.ApplicationId.ToString()"
-        class="searchable-element searchable-element-title"
-        category-filter-value="@Model.CategoryFilter"
-        topic-filter-value="@Model.TopicFilter">
+@{ var selected = Model.Selected ? @"selected=""selected""" : ""; }
+
+<!option value="@Model.ApplicationId.ToString()"

```

Comment:
I haven't seen an `!` used in an HTML tag before, except for with DOCTYPE. What does it do?

Reply:
I don't know honestly. I tried googling this but I didn't find anything useful. One difference that I noticed is that without it I get an error saying `The tag helper 'option' must not have C# in the element's attribute declaration area`, so we couldn't use `@selected` as an attribute. I pushed a work around to this that seems to have the same functionality, but without the `!`, I think it's better than having things we don't understand.

@SteveJacksonSoft  or maybe @simonbrent Can you explain what `!` does in an HTML tag?